content/learn/contribute/helping-out/creating-examples.md

```diff
@@ -5,22 +5,51 @@ insert_anchor_links = "right"
 Use a generous sprinkling of keywords in your description: these are commonly used to search for a specific example.
 See the [example style guide](#style-guide) to help make sure the style of your example matches what we're already using.
 
-More complex demonstrations of functionality are also welcome, but these should be submitted to [`bevy-assets`](https://github.com/bevyengine/bevy-assets).
+## Types of examples
+
+[Examples in Bevy](https://github.com/bevyengine/bevy/tree/main/examples) in Bevy can be categorized into three distinct categories,
+each of which have their own goals and subfolder:
+
+1. API examples: clearly demonstrating a single feature or group of closely related small features.
+2. Usage examples: teach users opinionated patterns for accomplishing common game development tasks.
+3. Small games: focused on integrating these patterns into a cohesive whole while demonstrating how to structure larger projects.
+
+In addition, we have two additional categories which are treated as examples by `cargo`, but are not intended to teach users:
+
+1. Stress tests: uses extreme entity counts or otherwise unrealistic settings to allow us to readily detect performance changes in particular performance-sensitive hot paths.
+2. Testbeds: designed to test features and scenes that rely on graphical rendering or interactivity, in both an automated and manual fashion.
+
+Unlike examples, stress tests and test beds should be configurable, allowing Bevy contriubtors (and curious users) to see the impact of changes.
+
+## When should I add an example?
+
+Adding, maintaining and searching through examples has non-trivial costs: we need to think carefully about our selection.
+Each of our example categories has its own goals which should be discussed separately.
+
+A good API example is complementary to good API documentation, and should only be added to promote the discovery of important features, show API usage in context, or show off graphical, audible or interactive functionality that cannot be appropriately conveyed by written documentation alone.
+When possible, prefer documentation tests and module documentation over API examples: these are easier to keep up to date, more clearly scoped and better connected to the APIs in question.
+
+Usage examples tackle common tasks that might be encountered by veterans to game development (or CAD or...) that are trying to translate existing concepts or workflows into Bevy's APIs.
+These should be focused on realism, building out the minimum required to demonstrate that they actually solve the problem at hand.
+Usage examples can be more opinionated, and may demonstrate multiple possible approaches to the same problem, explaining the tradeoffs involved.
+
+Game examples are much more expensive to maintain, and should be used sparingly. Each game example should demonstrate Bevy's capabilities in a new genre or style of game, while being simple enough for new Bevy users to follow and get excited about.
```

Comment:
How much should game examples repeat important information that is already provided elsewhere? For example, every first person game should imo have a [separate view model and world model](https://bevyengine.org/examples/camera/first-person-view-model/) for accessibility reasons, since Bevy cares a lot about accessibility.
Is it enough to add a comment on that part of a game example that says "see this example an explanation of why we are doing this"? Or should it try to be entirely self-contained?

Reply:
Cross-linking in various forms is fine :)